src/components/dialog/styles/dialog.d.ts

```diff
@@ -1,5 +1,10 @@
-export const main: string;
-export const underlay: string;
-export const title: string;
-export const content: string;
-export const close: string;
+export default {
+	classes: {
+		main: 'main',
```

Comment:
Was this generated using grunt-dojo2? I'm confused by this. It's in a `.d.ts` file, but is an actual module. It also doesn't look like the module maps generated class names? I'm probably missing something.